proto/dwtools/abase/l0.test/Buffer.test.s

```diff
@@ -11086,11 +11086,187 @@ function bufferRetype( test )
 
 function bufferRetypeWithOffset( test )
 {
-  test.case = 'F64x from BufferRaw with offset to U16x';
-  var raw = new BufferRaw( 32 );
-  var src = new F64x( raw, 0, 4 );
+  /* BufferNode */
+
+  if (Config.interpreter === 'njs') {
+
+    test.case = 'BufferNode from raw to U8x';
+    var raw = new U8x( [ 1, 2, 3, 4, 5, 6, 7, 8 ] ).buffer;
+    var src = new BufferNode.from( raw );
+    var got = _.bufferRetype( src, U8x );
+    var expected = new U8x( [ 1, 2, 3, 4, 5, 6, 7, 8 ] );
+    test.identical( got, expected );
+
+    test.case = 'BufferNode from raw to I8x';
+    var raw = new U8x( [ 1, 2, 3, 4, 5, 6, 7, 8 ] ).buffer;
+    var src = new BufferNode.from( raw );
+    var got = _.bufferRetype( src, I8x );
+    var expected = new I8x( [ 1, 2, 3, 4, 5, 6, 7, 8 ] );
+    test.identical( got, expected );
+
+    test.case = 'BufferNode from raw to F32x';
+    var raw = new U8x( [ 1, 2, 3, 4, 5, 6, 7, 8 ] ).buffer;
+    var src = new BufferNode.from( raw );
+    var got = _.bufferRetype( src, F32x );
+    var expected = new F32x( [ 1.539989614439558e-36, 4.063216068939723e-34 ] );
+    test.identical( got, expected );
+
+    test.case = 'BufferNode from raw to U64x';
+    var raw = new U8x( [ 1, 2, 3, 4, 5, 6, 7, 8 ] ).buffer;
+    var src = new BufferNode.from( raw );
+    var got = _.bufferRetype( src, U64x );
+    var expected = new U64x( [ ] );
+    test.identical( got, expected );
+  }
+
+  /* BufferView */
+
+  test.case = 'BufferView from raw to U8x';
+  var raw = new U8x( [ 1, 2, 3, 4, 5, 6, 7, 8 ] ).buffer;
+  var src = new BufferView( raw );
+  var got = _.bufferRetype( src, U8x );
+  var expected = new U8x( [ 1, 2, 3, 4, 5, 6, 7, 8 ] );
+  test.identical( got, expected );
+
+  test.case = 'BufferView from raw to I16x';
+  var raw = new U8x( [ 1, 2, 3, 4, 5, 6, 7, 8 ] ).buffer;
+  var src = new BufferView( raw );
+  var got = _.bufferRetype( src, I16x );
+  var expected = new I16x( [ 513, 1027, 1541, 2055 ] );
+  test.identical( got, expected );
+
+  test.case = 'BufferView from raw to F32x';
+  var raw = new U8x( [ 1, 2, 3, 4, 5, 6, 7, 8 ] ).buffer;
+  var src = new BufferView( raw );
+  var got = _.bufferRetype( src, F32x );
+  var expected = new F32x( [ 1.539989614439558e-36, 4.063216068939723e-34 ] );
+  test.identical( got, expected );
+
+  test.case = 'BufferView from raw to U64x';
+  var raw = new U8x( [ 1, 2, 3, 4, 5, 6, 7, 8 ] ).buffer;
+  var src = new BufferView( raw );
+  var got = _.bufferRetype( src, U64x );
+  var expected = new U64x( [] );
+  test.identical( got, expected );
+
+  /* BufferTyped */
+
+  /* I8x */
+
+  test.case = 'I8x from raw to with offset U8x';
+  var raw = new U8x( [ 1, 2, 3, 4, 5, 6, 7, 8 ] ).buffer;
+  var src = new I8x( raw, 1, 7 );
+  var got = _.bufferRetype( src, U8x );
+  var expected = new U8x( [ 2, 3, 4, 5, 6, 7, 8 ] );
+  test.identical( got, expected );
+
+  test.case = 'I8x from raw with offset to U16x';
```

Comment:
Please, change description. 

To make more informative descriptions use test groups:
```js
  test.open( 'src - I8x' );

  test.case = 'to U16x, src use section from 2 to end of raw'; // it's more informative, but not ideal. Please change description
  var raw = new U8x( [ 1, 2, 3, 4, 5, 6, 7, 8 ] ).buffer;
  var src = new I8x( raw, 2, 6 );
  var got = _.bufferRetype( src, I16x );
  var expected = new I16x( [ 1027, 1541, 2055 ] );
  test.identical( got, expected )

// other test cases

  test.close( 'src - I8x' );
```

Reply:
Maybe, also i should group bufferRetype  the same?